purchases/src/defaults/kotlin/com/revenuecat/purchases/IntentExtensions.kt

```diff
@@ -3,7 +3,6 @@ package com.revenuecat.purchases
 import android.content.Intent
 
 @ExperimentalPreviewRevenueCatPurchasesAPI
-@JvmSynthetic
-fun Intent.parseAsWebPurchaseRedemption(): WebPurchaseRedemption? {
-    return Purchases.parseAsWebPurchaseRedemption(this)
-}
+val Intent.asWebPurchaseRedemption: WebPurchaseRedemption?
```

Comment:
I like the name! But it does feel to me that this should be a function rather than a property. We are converting the object, so something is "happening", and a `WebPurchaseRedemption` is not really a property of the broad `Intent` type. Just my 2 cents. What do you think?

Reply:
Hmm I agree with you... I'm just doubting since it's a property in iOS... I guess we don't really need to copy it exactly though, so I can make the change 👍

Reply:
We can make it a function in iOS too 😈

Reply:
Ah yes, let's stay consistent with what's there already. Thanks for checking!